docs/high/dataset.rst

```diff
@@ -48,6 +50,71 @@ they will override ``data.shape`` and ``data.dtype``.  It's required that
 in ``data.shape``, and that (2) it's possible to cast ``data.dtype`` to
 the requested ``dtype``.
 
+.. _dataset_slicing:
+
+Reading & writing data
+----------------------
```

Comment:
Possibly it needs a rebase so these changes which are already in master don't show up as part of the PR.

Reply:
Done